content/applications/marketing/marketing_automation/getting_started/testing_running.rst

```diff
@@ -0,0 +1,42 @@
+Testing & running a campaign
+============================
+
+Odoo gives users the ability to test marketing campaigns (and mailings) before officially running
+them.
+
+First, open the :guilabel:`Marketing Automation` application and click on the campaign. Make sure
+the campaign already has activities configured on it. To start a test, click the :guilabel:`Launch a
+Test` button at the top of the template form.
+
+.. image:: testing_running/launch-test.png
```

Comment:
This image isn't helpful unless it shows other UI elements around it, otherwise we're saying "randomly look for THIS button". Would retake with other UI elements surrounding it and then highlight the button with red box.